Composer/packages/client/src/components/BotRuntimeController/BotControllerMenu.tsx

```diff
@@ -76,14 +77,24 @@ const tableColumns: IColumn[] = [
       );
     },
   },
+  {
+    key: 'webchat-viewer',
+    name: '',
+    minWidth: 130,
+    maxWidth: 130,
+    isRowHeader: true,
+    onRender: ({ projectId, isRootBot }) => {
+      return <OpenWebChatButton isRoot={isRootBot} projectId={projectId} />;
+    },
+  },
   {
     key: 'emulator',
     name: '',
     minWidth: 150,
     maxWidth: 150,
     isRowHeader: true,
-    onRender: ({ projectId, isRoot }) => {
-      return <OpenEmulatorButton isRoot={isRoot} projectId={projectId} />;
+    onRender: ({ projectId, isRootBot }) => {
```

Comment:
nit for the future improvements: if the detailslist used here is only showing one line of data, we should consider using something like CommandBar. DetailsList is overkill. @srinaath

Reply:
Got it. Will backlog this item.

Reply:
Got it. Will backlog this item.